File: src/main/java/com/wanderers/hotelier_webservices/server/dto/AccommodationDto.java
package com.wanderers.hotelier_webservices.server.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class AccommodationDto {
    private Integer id;
    private String hotelierId;
    private String name;
    private Integer rating;
    private Category category;
    private String city;
    private String state;
    private String country;
    private String zipCode;
    private String address;
    private String image;
    private Integer reputation;
    private ReputationBadge reputationBadge;
    private Integer price;
    private Integer availability;
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/server/dto/AccommodationDto.kt
package com.wanderers.hotelier_webservices.server.dto

import com.wanderers.hotelier_webservices.rest.model.ReputationBadgeEnum

data class AccommodationDto(
    var id: Int?,
    var hotelierId: String,
    var name: String,
    var rating: Int,
    var category: String,
    var city: String,
    var state: String,
    var country: String,
    var zipCode: String,
    var address: String,
    var image: String,
    var reputation: Int,
    var reputationBadge: ReputationBadgeEnum?,
    var price: Int,
    var availability: Int
)